src/Mentat.UI/Models/Card.cs

```diff
@@ -8,6 +8,16 @@ namespace Mentat.UI.Models
     public class Card
     {
         public Object _id { get; set; }
+        public string subject { get; set; }
```

Comment:
These public properties need to be renamed to follow the project's coding standards(PascalCaseProperties, no underscores allowed). Remember to also update anywhere the properties are referenced.